app/uk/gov/hmrc/fhregistrationfrontend/connectors/EmailVerificationConnector.scala

```diff
@@ -86,6 +86,6 @@ class DefaultEmailVerificationConnector @Inject()(
     }
 
     val url = s"$emailVerificationBaseUrl/verification-requests"
-    http.POST[EmailVerificationRequest, Boolean](url, request)
+    http.POST[EmailVerificationRequest, Boolean](url, request).map(_ => true)
```

Comment:
This page should not have been included, correct?  This change should be for local branch only?

Reply:
oops, missed it

Reply:
reverted